src/index.ts

```diff
@@ -445,7 +465,9 @@ export class Bigtable {
       {},
       baseOptions,
       {
-        servicePath: customEndpointBaseUrl || defaultBaseUrl,
+        servicePath:
+          customEndpointBaseUrl ||
+          `bigtable.${getDomain(options.BigtableClient)}`,
```

Comment:
What would you think about adding `pathPrefix` as an argument to getDomain (and maybe renaming the method if needed), instead of manually building the string here? I think that would be a bit easier to read, and seems more natural to me since the domain string isn't useful on its own

Reply:
Yup. That would be fine.